src/service.ts

```diff
@@ -41,7 +41,7 @@ function argsToOptions(args: string[]) {
 
   for (const arg of args) {
     let [key, rawValue] = arg.replace(/^-+/, "").split("=", 2);
```

Comment:
`.split('=', 2)` also causes problem for value with `=` in it.

<img width="227" alt="image" src="https://user-images.githubusercontent.com/8050659/223205028-4746d7d8-2176-48db-8be3-a86d4a7cf189.png">

Although for prettier it doesn't matter, because it doesn't accept any value with `=` in it.

But the previous code (`...valueParts` and `valueParts.join('=')` was more robust for parsing args.

Reply:
Otherwise, looks good 👍🏼

Reply:
<img src="https://media.giphy.com/media/3WmWdBzqveXaE/giphy.gif">

I didn't read the docs and assumed that limit behaved like in other languages. I'll just revert my other PR then.